src/ConfigFactory.Core/ConfigModule.cs

```diff
@@ -74,16 +74,16 @@ private static T Load()
 
     public virtual void Reset()
     {
-        IConfigModule config = Load();
-        foreach ((var name, (var property, _)) in Properties) {
+        var config = Load();
+        foreach (var (name, (property, _)) in Properties) {
             property.SetValue(Shared, config.Properties[name].Property.GetValue(config));
         }
     }
 
     protected virtual void Validate<TProperty>(Expression<Func<TProperty>> property, Func<TProperty?, bool> validation,
         string? invalidErrorMessage = null, string? validationFailureColor = null, string? validationSuccessColor = null)
     {
-        PropertyInfo propertyInfo = (PropertyInfo)((MemberExpression)property.Body).Member;
+        var propertyInfo = (PropertyInfo)((MemberExpression)property.Body).Member;
```

Comment:
Prefer explicit type names

```suggestion
        PropertyInfo propertyInfo = (PropertyInfo)((MemberExpression)property.Body).Member;
```

Reply:
Sure, no problem :)